layers/meta-resin-allwinner/recipes-kernel/rtl8189fs/rtl8189fs/0001-Use-modules_install-as-wanted-by-yocto.patch

```diff
@@ -0,0 +1,29 @@
+From 8e13d9ad5e31ece37f281fa16801dda9d2ffb345 Mon Sep 17 00:00:00 2001
```

Comment:
Can you change the commit log to show the component

> rtl8189fs: <Short description here>

instead of using the patch name, please?

Reply:
done